webapi/CopilotChat/Controllers/BotController.cs

```diff
@@ -254,6 +254,9 @@ private async Task<Bot> CreateBotAsync(IKernel kernel, Guid chatId)
         ChatSession chat = await this._chatRepository.FindByIdAsync(chatIdString);
         bot.ChatTitle = chat.Title;
 
+        // get the system description
+        bot.SystemDescription = chat.SystemDescription;
```

Comment:
I don't think we have a concrete definition for persona yet. It could include the system description (meta prompt), and the cognitive architecture, etc. We will stick with system description for now because it aligns with the rest of the prompt definitions.